apps/database/src/models/Purchase.ts

```diff
@@ -19,7 +19,7 @@ export class Purchase extends Model {
         enum: ["github", "stripe"],
       },
       endDate: { type: ["string", "null"] },
-      startDate: { type: ["string", "null"] },
+      startDate: { type: ["string"] },
```

Comment:
Nop
<img width="384" alt="Screenshot 2023-04-27 at 14 34 32" src="https://user-images.githubusercontent.com/266302/234863321-ceeab5fe-c309-412f-badd-85834486fef1.png">

Reply:
Model reflects the database, if you add a `NOT NULL` constraint then it is OK for me!

Reply:
Good to know! For me model was an abstraction layer used to add soft restrictions.